clients/venice-client/src/main/java/com/linkedin/venice/fastclient/stats/FastClientStats.java

```diff
@@ -96,6 +96,7 @@ private FastClientStats(MetricsRepository metricsRepository, String storeName, R
         return System.currentTimeMillis() - this.cacheTimeStampInMs;
       }
     }, "metadata_staleness_high_watermark_ms"));
+    this.multiKeyFanoutSizeSensor = registerSensor("multi_key_fanout_size", new Avg(), new Max());
```

Comment:
Shall we just called it `fanout_size` as the request type will be the prefix of the stat name?